### DISCUSSION TITLE
Secondary `slug` field without `name`s

### DISCUSSION THREAD

**Discussion by tyteen4a03** [Thinkmill/keystatic#887] [Ideas]
The `slug` field looks very nice. I would like to add secondary `slug` fields that generate nanoIDs, but there's no way of foregoing the `name` field.

It would be great if `name` object can be optional for non-primary `slug` fields.

**Comment by AliMejbar** [comment]
I up this, sometimes I also prefer other fields like a date or datetime to be linked with the slug
